content/docs/examples/advanced-gateways/egress_sni_monitoring_and_policies/index.md

```diff
@@ -0,0 +1,367 @@
+      namespace: istio-system
+    spec:
+      value: connection.requested_server_name
+    ---
+    # Rule to check access to *.wikipedia.org
+    apiVersion: "config.istio.io/v1alpha2"
+    kind: rule
+    metadata:
+      name: check-wikipedia-access
+      namespace: istio-system
+    spec:
+      match: source.labels["app"] == "istio-egressgateway-with-sni-proxy" && destination.labels["app"] == ""
+      actions:
+      - handler: wikipedia-checker.listchecker
+        instances:
+          - requested-server-name.listentry
+    EOF
+    {{< /text >}}
+
+1.  Send an HTTPS request to the blacklisted [Wikipedia in English](https://en.wikipedia.org):
+
+    {{< text bash >}}
+    $ kubectl exec -it $SOURCE_POD -c sleep -- sh -c 'curl -v https://en.wikipedia.org/wiki/Main_Page'
+    ...
+    curl: (35) Unknown SSL protocol error in connection to en.wikipedia.org:443
+    command terminated with exit code 35
+    {{< /text >}}
+
+    Access to the Wikipedia in English is blocked according to the policy you defined.
+
+1.  Send HTTPS requests to some other Wikipedia sites, for example [https://es.wikipedia.org](https://es.wikipedia.org) and
+    [https://de.wikipedia.org](https://de.wikipedia.org):
+
+    {{< text bash >}}
+    $ kubectl exec -it $SOURCE_POD -c sleep -- sh -c 'curl -s https://es.wikipedia.org/wiki/Wikipedia:Portada | grep -o "<title>.*</title>"; curl -s https://de.wikipedia.org/wiki/Wikipedia:Hauptseite | grep -o "<title>.*</title>"'
+    <title>Wikipedia, la enciclopedia libre</title>
+    <title>Wikipedia – Die freie Enzyklopädie</title>
+    {{< /text >}}
+
+    Access to Wikipedia sites in other languages is allowed, as expected.
+
+### Cleanup of monitoring and policy enforcement
+
+{{< text bash >}}
+$ kubectl delete rule handle-wikipedia-access check-wikipedia-access -n istio-system
+$ kubectl delete logentry egress-access -n istio-system
+$ kubectl delete stdio egress-access-logger -n istio-system
+$ kubectl delete listentry requested-server-name -n istio-system
+$ kubectl delete listchecker wikipedia-checker -n istio-system
+{{< /text >}}
+
+## Monitor the SNI and the source identity, and enforce access policies based on them
+
+Since you enabled mutual TLS between the sidecar proxies and the egress gateway, you can monitor the [service identity](/docs/concepts/what-is-istio/#citadel) of the applications that access external services, and enforce policies
+based on the identities of the source of traffic.
+In Istio on Kubernetes, the identities are based on
+[Service Accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/). In this
+subsection, you deploy two _sleep_ containers, `sleep-us` and `sleep-canada` under two service accounts,
+`sleep-us` and `sleep-canada`, respectively. Then you define a policy that allows applications with the `sleep-us`
+identity access the English and the Spanish versions of Wikipedia, and services with `sleep-canada` identity access the
```

Comment:
s/access the/to access the/g # two places

Reply:
Addressed in d3a0846945a805ee428240dc8375bbb72ef7718e